.github/workflows/test.yaml

```diff
@@ -59,6 +59,7 @@ jobs:
         working-directory: ./src/FunctionApps/${{matrix.function-to-test}}
         run: |
           pip install -U pip
+          sed -i "\|prime-public-health-data-infrastructure@|s|main|$GITHUB_HEAD_REF|" requirements.txt
```

Comment:
:guitar:

Reply:
This is interesting because I trying it using Git Bash on Windows and it doesn't appear to be changing anything. But I trust that this is working, and we'll find out if it doesn't come test time. 😅

Reply:
Yup, I see my confusion now. I misinterpreted the `prime-public-health-data-infrastructure@` to be the string being replaced, as opposed to a preceding string to help identify where `main` should be replaced with `$GITHUB_HEAD_REF`. This all makes sense now.